### DISCUSSION TITLE
[求助]怎么登陆啊

### DISCUSSION THREAD

**Discussion by shidijun** [share121/inter-knot#5025] [绝区零]
怎么办啊 一改为用户接口不仅登不上 还给我出这个
<img src="https://github.com/user-attachments/assets/489f820b-160d-44aa-8934-0d736712f9a0">

**Comment by DogOfFurina114514** [comment]
用**VPN**试试

**Reply by shidijun** [reply]
具体怎么用啊qwq

**Reply by Breeze520** [reply]
下了直接连呗，不然怎么用

**Reply by DogOfFurina114514** [reply]
> 具体怎么用啊qwq

在插件市场搜Hoxx VPN Proxy

**Comment by March-777th** [comment]
改用客户端试试[inter_knot](https://github.com/share121/inter-knot/releases/download/v2.16.7%2B34/inter_knot_windows_x64_boxed.exe)
